src/groups/special_linear.jl

```diff
@@ -78,8 +78,9 @@ function get_embedding(M::SpecialLinear{Tuple{Int},𝔽}) where {𝔽}
     return GeneralLinear(n, 𝔽; parameter=:field)
 end
 
-inverse_translate_diff(::SpecialLinear, p, q, X, ::LeftForwardAction) = X
-inverse_translate_diff(::SpecialLinear, p, q, X, ::RightBackwardAction) = p * X / p
+# note: this implementation is not optimal
```

Comment:
Could we write that as `(p * X)  / p` to avoid the inv?

Reply:
I don't think you can avoid the `inv` because `X` is a matrix. What I meant is that one can probably avoid some unnecessary allocations (the allocated `Y` is not used except in the end with a `copyto`).

Reply:
I did not check that yet but I can, it just reminded me a lot of the $p^{1/2}Xp^{-1/2}$ and such we do on SPDs, where `/`is exactly what we do; and sure both `p` and `X` are matrices.

Reply:
I removed `inv`.